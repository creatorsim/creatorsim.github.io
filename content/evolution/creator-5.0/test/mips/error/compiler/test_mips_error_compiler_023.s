
#
# Creator (https://creatorsim.github.io/creator/)
#

 .data
 	.align 2
 	example:	.byte 	23
   example2:	.half	
    