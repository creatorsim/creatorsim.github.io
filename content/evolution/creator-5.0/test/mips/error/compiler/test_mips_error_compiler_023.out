

[E09] Error: Incorrect amount of arguments, expected at least `1` but found `0`
   ╭─[ assembly:9:20 ]
   │
 9 │    example2:    .half
   │                       ┬  
   │                       ╰── This directive has `0` arguments
   │ 
   │ Help: Consider adding the missing `1` argument
───╯

Not executed
keyboard[0x0]:''; display[0x0]:''; 

